doc/user_manual/chSpecialElements.tex

```diff
@@ -783,22 +781,22 @@ \subsection{Examples}
 The \texttt{FUN} type \texttt{COSF\_RIPP} is provided for exactly this purpose, and provides an exact replacement.
 As an example, the \texttt{RIPP} block in the SixTest test-case prob1 looks like (slightly reduced in size):
 \begin{cverbatim}
-RIPPLE OF POWER SUPPLIES------------------------------------------------
-         dmqx1f50l5+2      3.2315D-10    224.9
-         dmqx2af50l5+2    -3.2315D-10    224.9
-         dmqx1f10mel5+2    2.5246D-16    0.0011245
+RIPPLE OF POWER SUPPLIES
+  dmqx1f50l5+2      3.2315D-10    224.9
+  dmqx2af50l5+2    -3.2315D-10    224.9
+  dmqx1f10mel5+2    2.5246D-16    0.0011245
 NEXT
 \end{cverbatim}
 This can be replaced by the following:
 \begin{cverbatim}
 DYNK
-NOFILE
-FUN RIPP-dmqx1f50l5+2 COSF_RIPP 3.2315D-10 224.9 0.0
-SET dmqx1f50l5+2 average_ms RIPP-dmqx1f50l5+2 1 -1 0
-FUN RIPP-dmqx2af50l5+2 COSF_RIPP -3.2315D-10 224.9 0.0
-SET dmqx2af50l5+2 average_ms RIPP-dmqx2af50l5+2 1 -1 0
-FUN RIPP-dmqx1f20kl5+2 COSF_RIPP 2.5246D-12 0.56225 0.0
-SET dmqx1f20kl5+2 average_ms RIPP-dmqx1f20kl5+2 1 -1 0
+  DYNKSETS
```

Comment:
Here `NOFILE` has been replaced by `DYNKSETS`, inverting the logic. And unless you're debugging FUN:RIPP, you probably don't want a humongous dynksets file for this!